Testscripts/Linux/perf_apache.sh

```diff
@@ -0,0 +1,255 @@
+		Run_SSHCommand "${server}" "pkill -f mpstat"
+		Run_SSHCommand "${server}" "pkill -f sar"
+		Run_SSHCommand "${server}" "pkill -f vmstat"
+		Run_SSHCommand "${server}" "${dstat_cmd} -dam" > "${log_folder}/${current_concurrency}.dstat.server.log" &
+		Run_SSHCommand "${server}" "${sar_cmd} -n DEV 1" > "${log_folder}/${current_concurrency}.sar.server.log" &
+		Run_SSHCommand "${server}" "${mpstat_cmd} -P ALL 1" > "${log_folder}/${current_concurrency}.mpstat.server.log" &
+		Run_SSHCommand "${server}" "${vmstat_cmd} 1" > "${log_folder}/${current_concurrency}.vmstat.server.log" &
+
+		pkill -f dstat
+		pkill -f mpstat
+		pkill -f sar
+		pkill -f vmstat
+		${dstat_cmd} -dam > ${log_folder}/${current_concurrency}.dstat.server.log &
+		${sar_cmd} -n DEV 1 > ${log_folder}/${current_concurrency}.sar.server.log &
+		${mpstat_cmd} -P ALL 1 > ${log_folder}/${current_concurrency}.mpstat.server.log &
+		${vmstat_cmd} 1 > ${log_folder}/${current_concurrency}.vmstat.server.log &
+
+		Run_ab ${current_concurrency}
+
+		Run_SSHCommand "${server}" "pkill -f dstat"
+		Run_SSHCommand "${server}" "pkill -f mpstat"
+		Run_SSHCommand "${server}" "pkill -f sar"
+		Run_SSHCommand "${server}" "pkill -f vmstat"
+		pkill -f dstat
+		pkill -f mpstat
+		pkill -f sar
+		pkill -f vmstat
+
+		sleep 10
+		apache_log_file="${log_folder}/${current_concurrency}.${testName}.bench.log"
+		WebServerVersion=$(grep "Server Software:" "$apache_log_file" | head -n1 | tr ":" " " | awk '{print $NF}')
+		concurrency=$(grep "Concurrency Level:" "$apache_log_file" | head -n1 | tr ":" " " | awk '{print $NF}')
+		documentLength=$(grep "Document Length" "$apache_log_file" | head -n1 | tr ":" " " | awk '{print $3}')
+		requests=$(grep "Complete requests:" "$apache_log_file" | tr ":" " " | awk '{print $NF}' | awk 'BEGIN{} {sum+=$1} END {print sum}')
+		numOfAb=$(grep "Complete requests:" "$apache_log_file" | wc -l)
+		req_sec=$(grep "Requests per second:" "$apache_log_file" | tr ":" " " | awk '{print $4}' | awk 'BEGIN{} {sum+=$1} END {print sum}')
+		transfer=$(grep "Transfer rate:" "$apache_log_file" | tr ":" " " | awk '{print $3}' | awk 'BEGIN{sum=0.0} {sum+=$1} END {print float sum}')
+		meanTime=$(grep "Total:" "$apache_log_file" | tr ":" " " | awk '{print $3}' | awk 'BEGIN{} {sum+=$1} END {print sum/NR}')
+
+		LogMsg "Test Results: "
+		LogMsg "---------------"
+		LogMsg "WebServerVersion: $WebServerVersion"
+		LogMsg "Test Concurrency: $current_concurrency"
+		LogMsg "Number Of AbInstances: $numOfAb"
+		LogMsg "Concurrency Per AbInstance: $concurrency"
+		LogMsg "Document Length: $documentLength"
+		LogMsg "Complete Requests: $requests"
+		LogMsg "Requests Per Sec: $req_sec"
+		LogMsg "Transfer Rate(KBps): $transfer"
+		LogMsg "Mean Connection Times (ms): $meanTime"
+
+		echo "$WebServerVersion,$current_concurrency,$numOfAb,$concurrency,$documentLength,$requests,$req_sec,$transfer,$meanTime" >> "${result_file}"
+
+		LogMsg "sleep 50 seconds..."
+		sleep 50
```

Comment:
Same as above comments.

Reply:
This is inherited from lis-test repo. I don't know why is so long sleep. When I deleted this sleep, the test result was the same. So I deleted this sleep.